src/nvim/charset.c

```diff
@@ -886,14 +902,14 @@ int vim_isfilec_or_wc(int c)
   return vim_isfilec(c) || c == ']' || path_has_wildcard(buf);
 }
 
-/// return TRUE if 'c' is a printable character
+/// return true if 'c' is a printable character
 /// Assume characters above 0x100 are printable (multi-byte), except for
 /// Unicode.
 ///
 /// @param c
 ///
-/// @return TRUE if 'c' a printable character.
-int vim_isprintc(int c)
+/// @return true if 'c' a printable character.
+bool vim_isprintc(int c)
```

Comment:
If you are already modifying this line then I think you can add `FUNC_ATTR_PURE` here as well.